src/main/resources/com/ncc/aif/ontologies/SeedlingOntology

```diff
@@ -4,18 +4,17 @@
 @prefix schema: <http://schema.org/> .
 @prefix ldcOnt: <https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/SeedlingOntology#> .
 @prefix olia:  <http://purl.org/olia/system.owl#> .
-@prefix aidaDomainCommon: <https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/AidaDomainOntologiesCommon#> .
+@prefix aidaDomainCommon: <https://tac.nist.gov/tracks/SM-KBP/2019/ontologies/AidaDomainOntologiesCommon#> .
```

Comment:
I have an email in to Hoa about this, but I believe we shouldn't change SeedlingOntology at all.
If we do change it, then we should have a 2019 version of Seedling and thus change references to Seedling everywhere.  Thoughts?

Reply:
Yeah, we should probably just update everything to 2019 as it's a reference to SM-KBP and not the ontologies

Reply:
To update this comment: at 1:42p EDT, Hoa replied to the email stating that she WOULD maintain the 2018 directory of ontologies so we can keep the Seeding ontology reference as 2018